### PR TITLE
Fix the use of SuiteSparse from vcpkg

### PR DESCRIPTION
**Pull request by lrineau**
Recent version of `suitesparse-metis-for-windows` no longer provide a `USE_SuiteSparse` CMake variable. I got that bug because I tried something from Linux version of `vcpkg`.

@afabri You may want to check because you wrote those lines of CMake, to support Suitesparse on Windows, a few years back.

### CODE REVIEW

**Comment by afabri** [comment]
@soesau can you please take care of this as you used vcpk recently.

File: Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt [hunk 1]
```diff
@@ -38,7 +36,7 @@ if(TARGET CGAL::Eigen3_support)
       message(STATUS "NOTICE: The example `orbifold.cpp` will be compiled without the Sparsesuite library and UmfPack. Try setting SuiteSparse_UMF_INCLUDE_DIR and at least one of SuiteSparse_UMFPACK_LIBRARY_RELEASE and SuiteSparse_UMFPACK_LIBRARY_DEBUG to you UMFPACK installation.")
     endif()
   else(SuiteSparse_FOUND)
-    message(STATUS "NOTICE: The example `orbifold.cpp` will be compiled without the Sparsesuite library.")
+    message(STATUS "NOTICE: The example `orbifold.cpp` will not be compiled without the Sparsesuite library.")
```
Comment (albert-github) [inline Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt:39]: Sparsesuite -> SuiteSparse ?

Comment (lrineau) [reply Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt]: I fixed a few typos inb cf31d6ed65caa70038a8655864c9a20806f0dd6. There is still one typo in the name of the macro `CGAL_SMP_USE_SPARSESUITE_SOLVERS` but I do not know what I may break if I rename it. It seems an internal detail.

Comment (afabri) [reply Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt]: If it is not documented I would just change it.

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-223

**Pull request merged**